From 4ac454f3d239df546f92056d70dfef1257bbda7a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Mon, 12 Feb 2018 11:05:08 +0100
Subject: [PATCH] Do not link to nsl library
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Net-Patricia code does not uses any symbols from nsl library, there is
no need to link to it. Actually it causes build failures on systems
that moved nsl library from glibc to separate project
<https://github.com/thkukuk/libnsl>.

CPAN RT#124088

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 Makefile.PL | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index acd0918..c023e43 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -5,7 +5,6 @@ use ExtUtils::MakeMaker;
 WriteMakefile(
     'NAME'	=> 'Net::Patricia',
     'VERSION_FROM' => 'Patricia.pm', # finds $VERSION
-    'LIBS'	=> ['-lnsl'],   # e.g., '-lm' 
     'DEFINE'	=> '',     # e.g., '-DHAVE_SOMETHING' 
     'INC'	=> '',     # e.g., '-I/usr/include/other' 
     'MYEXTLIB'	=> 'libpatricia/libpatricia$(LIB_EXT)',     # e.g., '-I/usr/include/other' 
-- 
2.13.6

